app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+

```

Comment:
Edit imports: add an empty line between them and make error imports like:
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)

There are no errors in the code block, it just seems to me that it will be more beautiful)))